src/node.cc

```diff
@@ -2681,6 +2681,9 @@ static void ProcessTitleSetter(Local<Name> property,
 static void EnvGetter(Local<Name> property,
                       const PropertyCallbackInfo<Value>& info) {
   Isolate* isolate = info.GetIsolate();
+  if (property->IsSymbol()) {
+    return info.GetReturnValue().SetUndefined();
```

Comment:
Calling `.SetUndefined()` explicitly doesn't hurt but it's not strictly necessary as it's the default return value.  We don't call it anywhere else.